app/controllers/patients_controller.rb

```diff
@@ -155,6 +155,9 @@ def create
     # Set the creator as the current user
     patient.creator = current_user
 
+    # Set the enrolled_isolation bool based on isolation bool value
+    patient.enrolled_isolation = patient.isolation
+
```

Comment:
I think we would prefer to do this on the patient model in a callback, like `before_create`. That way it happens for API and Import as well.